product_configurator_sale/models/sale.py

```diff
@@ -54,7 +54,9 @@ def reconfigure_product(self):
             model_name=wizard_model, extra_vals=extra_vals
         )
 
-    @api.depends()
+    @api.depends(
+        "config_session_id",
```

Comment:
Looks like it depends on more fields... no?

Reply:
Yes, but the price of the line should not change when every mentioned field changes, for instance: if the taxes on the product (`product_id.taxes_id`) change we don't want all the lines including that product to change their price, that would be unexpected I think.
I included only fields in the line (`product_id` is already _depended on_ in `super`).